### PR TITLE
[ISV-3730] creating script invoked by prow

### PR DESCRIPTION
**Pull request by ezopezo**

### CODE REVIEW

File: ci/prow/main.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+while :; do
+    current_time=$(date +%s)
+    elapsed_time=$((current_time - start_time))
+
+    if [[ $elapsed_time -ge $timeout ]]; then
+        echo "Timeout reached. Labels not found, check static tests logs."
+        exit 1
+    fi
+
+    API_RESPONSE=$(curl -s -H "Authorization: Bearer $GITHUB_TOKEN" "https://api.github.com/repos/$GITHUB_REPO/pulls/$PR_NUMBER/labels")
```
Comment (Allda) [inline ci/prow/main.sh]: Doesn't this expose the token in logs?

Comment (ezopezo) [reply ci/prow/main.sh]: This is just dummy script tested locally Ill simplify it for the testing purposes

File: ci/prow/main.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo "Prow script initiated."
+oc whoami
+echo "Prow complete."
```
Comment (Allda) [inline ci/prow/main.sh]: Please add the missing EOL

**Review by mporrato — APPROVED** [review]

File: ci/prow/main.sh (hunk 2 above)
Comment (ezopezo) [reply ci/prow/main.sh]: Done, thanks for noticing

**Review by Allda — APPROVED** [review]

**Pull request merged**